servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -2375,6 +2375,52 @@ public void deleteDomainTemplate(ResourceContext ctx, String domainName, String
         dbService.executeDeleteDomainTemplate(ctx, domainName, templateName, auditRef, caller);
     }
 
+    @Override
+    public DomainMetaStoreValidValuesList getDomainMetaStoreValidValuesList(ResourceContext ctx, String attributeName, String userName) {
```

Comment:
Do we have test cases for this new method?

Reply:
Added missing tests